Feature/add new grid view

Gustavinho opened the pull request:
# New grid view

<img width="1499" alt="Captura de Pantalla 2020-08-02 a la(s) 10 47 21 p m" src="https://user-images.githubusercontent.com/4228757/89144817-6cd23880-d514-11ea-9adf-07ca0a506197.png">

I added a new view based on the `TableView` class, this is to show data as a grid of cards, mostly to show images related to the data, I wrote the main features for this view, a couple of tests and its components

## What is included in this PR 
- Base grid view document file
- Tailwindcss updated to 1.5.2
- Changes with the alert to be displayed as a fixed-float element
- Base card component to be used by the grid view
- Dropdown component now has a small transition
- Base grid view layout
- GridView features
- Grid view tests (it doesn't need so much test cases because it is based on the TableView class)

fawzanm commented:
Hi @Gustavinho 

I've tried this PR in one of my existing apps. I faced a couple of issues. 

1. I couldn't get the pagination working. I've added the below snippet to the grid but it didn't render the pagination. 
```
    protected $paginate = 5;
```

2. The filter view had an issue with the content (probably a z-index issue
<img width="209" alt="Screenshot 2020-08-03 at 22 39 31" src="https://user-images.githubusercontent.com/2080551/89215955-981f5b00-d5da-11ea-81b9-f8f07816ed5e.png">

Other than that it looks great. The additional thing pop into my head is just future PRs. 

Let me know if you need any extra information. I'd be glad to share them.

Gustavinho commented:
> Hi @Gustavinho
> 
> I've tried this PR in one of my existing apps. I faced a couple of issues.
> 
> 1. I couldn't get the pagination working. I've added the below snippet to the grid but it didn't render the pagination.
> 
> ```
>     protected $paginate = 5;
> ```
> 
> 1. The filter view had an issue with the content (probably a z-index issue).
> 
> <img alt="Screenshot 2020-08-03 at 22 39 31" width="209" src="https://user-images.githubusercontent.com/2080551/89215955-981f5b00-d5da-11ea-81b9-f8f07816ed5e.png">
> 
> Other than that it looks great. The additional thing pop into my head is just future PRs.
> 
> Let me know if you need any extra information. I'd be glad to share them.

Gustavinho commented:
> > Hi @Gustavinho
> > I've tried this PR in one of my existing apps. I faced a couple of issues.
> > 
> > 1. I couldn't get the pagination working. I've added the below snippet to the grid but it didn't render the pagination.
> > 
> > ```
> >     protected $paginate = 5;
> > ```
> > 
> > 
> > 
> > 1. The filter view had an issue with the content (probably a z-index issue).
> > 
> > <img alt="Screenshot 2020-08-03 at 22 39 31" width="209" src="https://user-images.githubusercontent.com/2080551/89215955-981f5b00-d5da-11ea-81b9-f8f07816ed5e.png">
> > Other than that it looks great. The additional thing pop into my head is just future PRs.
> > Let me know if you need any extra information. I'd be glad to share them.

Yes thank you I already added the paginator, I totally forgot it, about the z-index, you might need to republish the assets, in that case, I'll add that note on the documentation
